common/ontology/apartment_go_2019.owl

```diff
@@ -524,7 +524,7 @@
     <!--  Maps furniture to rooms-->
     <owl:ObjectProperty rdf:about="apartment:locatedAt">
         <rdf:type rdf:resource="&owl;FunctionalProperty" />
-        <rdfs:domain rdf:resource="apartment:Object"/>
+        <rdfs:domain rdf:resource="apartment:Furniture"/>
```

Comment:
I would revert this change. The comment is what's wrong here. Having `Object` as a domain makes the property more general.

Reply:
I reverted the changed and removed `locatedInRoom`